### PR TITLE
chore: prepare v1

### PR DESCRIPTION
**Pull request by ananas-block**
preparation pr for refactor:
removed:
- psp-examples
- macro-circom
- psp system programs

commented:
- ci tests for the above and cli

### CODE REVIEW

**Comment by vadorovsky** [comment]
Please update `Cargo.toml`, macro-circom is still being included there

File: .github/workflows/light-sdk-tests.yml [hunk 1]
```diff
@@ -1,73 +1,73 @@
...
+#     name: light-sdk-tests
+#     if: github.event.pull_request.draft == false
+#     runs-on: buildjet-16vcpu-ubuntu-2204
+#     strategy:
+#       matrix:
+#         include:
+#           - name: sdk-tests-light-rpc-cli
+#             sub-tests: '[
+#               "@lightprotocol/cli:test",
+#               "@lightprotocol/prover.js:test"
```
Comment (SwenSchaeferjohann) [inline .github/workflows/light-sdk-tests.yml:31]: let's keep prover.js test

File: .github/workflows/light-sdk-tests.yml [hunk 2]
```diff
@@ -1,73 +1,73 @@
...
+#             ]'
+#           - name: circuit-tests
+#             sub-tests: '[
+#               "@lightprotocol/circuit-lib.js:test",
+#               "@lightprotocol/circuit-lib.circom:test"]'
```
Comment (SwenSchaeferjohann) [inline .github/workflows/light-sdk-tests.yml:36]: let's keep tests for circuitlib.js as well

File: .github/workflows/light-system-programs-tests.yml [hunk 3]
```diff
@@ -1,60 +1,60 @@
...
-      - reopened
-      - ready_for_review
+# on:
+#   push:
+#     branches:
+#       - main
+#     paths:
+#       - "zk.js/**"
+#       - "programs/**"
+#       - "light-circuits/**"
```
Comment (SwenSchaeferjohann) [inline .github/workflows/light-system-programs-tests.yml:8]: did we have a light-circuits dir? 👀

**Review by SwenSchaeferjohann — CHANGES_REQUESTED** [review]
as michal said e.g lint/fmt are still looking for macro circom which fails the respective tests

File: .github/workflows/light-system-programs-tests.yml (hunk 3 above)
Comment (ananas-block) [reply .github/workflows/light-system-programs-tests.yml:8]: that must have been a while ago

**Review by SwenSchaeferjohann — APPROVED** [review]
lgtm

**Pull request merged**